docs/transit-changes/README.md

```diff
@@ -1,133 +1,243 @@
+   - `update-transit-changes!` stores final results to db
+   `gtfs-change-result`, `detected-route-change` (For details refer to [Datamodel](#Datamodel))
+
+##### References
+   - https://github.com/finnishtransportagency/mmtis-national-access-point/blob/master/ote/src/clj/ote/tasks/gtfs.clj
+   - [Transit changes related db tables](#Datamodel)
    
-   **A differing week was found**  
-   If there is a different week in future, the alorighm starts comparing specific dates of the week in more depth per 
-   route.
-
-1. **Compare differing dates**  
-   After the process has found a week in future that has different traffic than current week, the first day is picked 
-   from the different week that starts the new traffic pattern. Then, the process picks the matching week day from
-   current week and compares the traffic of these two days in depth.
-   The in-depth comparison of the two differing days generates trip, stop sequence and stop time differences per route.
-    
-    1. **Trip differences**  
-       ![Trip differences example](trip-differences.png)
-      
-       The algorithm fetches trips of route for the current day and the different week day. The trip comparison works in the
-       following way. First, it tries to find the first common stop that is in every trip. Then the trips of the compared days
-       are combined using the common stop as a basis. This helps lining up the trip stop times next to each other so they
-       can be easily compared. In the merging process, time differences of the trip stop time pairs are computed.
-       Any compared trip stop time that is not within 30 minute time window causes the algorithm to mark the
-       trip either added or removed.
-      
-    1. **Stop sequence differences**  
-      Stop sequence differences are computed per compared trip pair. The algorithm detects stops that are added or
-      removed from a trip by comparing the stop sequences of the trips.
-      
-    1. **Stop time differences**  
-      Stop time differences are computed per compared trip pair. The departure time differences are computed for each 
-      matching stop pair.
- 
-
-
-### Notifications
-
-Notifications about newly detected transit changes are sent via email to transit authorities each day.
-
+#### Detection algorithm
+
+##### Important notes
+- Change detection is run periodically for the **latest imported** transit data packages.  
+- No history data is used from preceding imports or analyses. Latest package is analysed independently
+- Separate detection runs for the same service will detect same changes as the previous run, if the data is the same. 
+- Route traffic changes detected for current week when detection is run, will be reported on next week instead, except **added** and **ending** routes.
+- Thresholds
+  - Detection window: service's route data analyzed into future for: **120** days 
+  - Route end: route marked 'ending' if no traffic within next: **90** days
```

Comment:
These magic numbers might need more explanations. 
120 days is selected because transit operators can't provide data accurately enough to more than 120 days to the future.
90 days is selected because many of the transit operators can't provide data after summer break early enough.
14 days is selected because the detection window must be moved periodically but not too often.

Reply:
Indeed!
Such commenting could be put to the config file where these values will be moved and link there, to avoid duplicate maintenance.

Reply:
I'll do that in a separate PR however.